src/ghost-content/transparent.js

```diff
@@ -32,6 +33,21 @@ module.exports = ({ logger, projectLocation }) => {
       const filePath = path.join(folderPath, file)
       return fs.readFileAsync(filePath, 'utf-8')
     },
+
+    deleteFile: (folder, file) => {
+      const { folderPath } = normalizeFolder(folder)
+      const filePath = path.join(folderPath, file)
+      return Promise.fromCallback(cb => fs.unlink(filePath, cb))
+    },
+
+    directoryListing: (rootFolder, fileEndingPattern = '') => {
+      const flowsDir = normalizeFolder(rootFolder)
```

Comment:
This is obviously untested, `normalizeFolder` returns an object, see `const { folderPath } = normalizeFolder(folder)` above

Reply:
Good catch! Fixed that earlier but didn't push yet.